src/Mapbender/CoreBundle/Component/SecurityContext.php

```diff
@@ -220,8 +222,8 @@ public function isUserAnOperator($object)
      */
     public function isUserAllowedToCreate($object)
     {
-        //$oid = new ObjectIdentity('class', get_class($object));
-        return $this->isGranted(self::PERMISSION_CREATE, $object);
+        $identity = $this->getClassIdentity($object);

```

Comment:
The methods is static, so please `static::getClassItedentity($object)` instead.

Reply:
I actually prefer ```$this->``` whenever possible, because calling code doesn't need to change if / when the method changes to non-static (e.g. it needs to access logging or something else via container and can't stay static anymore).